extensions/wikia/WikiFactory/Close/maintenance.php

```diff
@@ -416,6 +413,38 @@ private function getDirTree( $dir ) {
 	}
 
 	/**
+	 * Remove DFS bucket of a given wiki
+	 *
+	 * @see PLATFORM-1700
+	 * @param int $cityId
+	 */
+	private function removeBucket( $cityid ) {
+		try {
+			$swift = \Wikia\SwiftStorage::newFromWiki( $cityid );
+			$this->log( sprintf( "Removing DFS bucket /%s%s", $swift->getContainerName(), $swift->getPathPrefix() ) );
+
+			// s3cmd --recursive del s3://BUCKET/OBJECT / Recursively delete files from bucket
+			$cmd = sprintf(
+				'sudo /usr/bin/s3cmd -c %s --recursive del s3://%s%s/',
```

Comment:
This is really hacky. Can you use something more native for PHP like [boto](https://github.com/boto/boto) for Python?

In case of any config changes or directory restructure I'm afraid we will forget about `const S3_CONFIG` and this logic.